osg-comanage-project-usermap.py

```diff
@@ -159,13 +179,15 @@ def parse_options(args):
 
     for op, arg in ops:
         if op == '-h': usage()
-        if op == '-u': options.user      = arg
-        if op == '-c': options.osg_co_id = int(arg)
-        if op == '-d': passfd            = int(arg)
-        if op == '-f': passfile          = arg
-        if op == '-e': options.endpoint  = arg
-        if op == '-o': options.outfile   = arg
-        if op == '-g': options.filtergrp = arg
+        if op == '-u': options.user       = arg
+        if op == '-c': options.osg_co_id  = int(arg)
+        if op == '-d': passfd             = int(arg)
+        if op == '-f': passfile           = arg
+        if op == '-e': options.endpoint   = arg
+        if op == '-o': options.outfile    = arg
+        if op == '-g': options.filtergrp  = arg
+        if op == '-t': options.minTimeout = float(arg)
+        if op == '-T': options.maxTimeout = float(arg)
```

Comment:
```suggestion
        if op == '-t': options.min_timeout = float(arg)
        if op == '-T': options.max_timeout = float(arg)
```

Reply:
I've incorporated this change in 80e513c